exasol/analytics/audit/columns.py

```diff
@@ -0,0 +1,50 @@
+from exasol.analytics.schema import (
+    decimal_column,
+    timestamp_column,
+    varchar_column,
+)
+
+
+class BaseAuditColumns:
+    TIMESTAMP = timestamp_column("LOG_TIMESTAMP", precision=3)
+    SESSION_ID = decimal_column("SESSION_ID", precision=20)
+    # RUN_ID must be obtained initially and remain unchanged during lifetime
+    # of AuditLogger. AuditLogger must inherit from QueryHandler and its first
+    # task is to get the RUN_ID.
+    # Proposed value: POSIX_TIME(SYSTIMESTAMP(9)) * 1000
+    RUN_ID = decimal_column("RUN_ID", precision=20)
+    ROWS_COUNT = decimal_column("ROWS_COUNT", precision=36)
+    # For ModifyQuery SPAN_TYPE is the Operation Name, e.g.
+    # CREATE_TABLE, INSERT, ERROR, COMMIT, CUSTOM:
+    SPAN_TYPE = varchar_column("SPAN_TYPE", size=128)
+    # SPAN_ID needs to be generated and provided by the creator of the
+    # AuditQuery, i.e. lower level query_handlers.
+    SPAN_ID = decimal_column("SPAN_ID", precision=32)
+    PARENT_SPAN_ID = decimal_column("PARENT_SPAN_ID", precision=32)
```

Comment:
What is "PARENT_SPAN_ID"?

Reply:
This concept is copied from https://opentelemetry.io/docs/concepts/signals/traces/.
There you have  `parent_id`.